solutions/instances/variables.tf

```diff
@@ -55,7 +55,7 @@ variable "provider_visibility" {
   description = "Set the visibility value for the IBM terraform provider. Supported values are `public`, `private`, `public-and-private`. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/guides/custom-service-endpoints)."
   type        = string
   # Defaulting this to public to workaround https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5977
-  default = "public"
+  default = "private"
```

Comment:
We can't set this to private yet. As per the above comment, there is a provider bug open when you try to create a log routing tenant and this is set to private: https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5977